ObjObsSAP.tex

```diff
@@ -581,19 +580,19 @@ \subsubsection{Standard output fields}
 
 \item {Exactly one field \textbf{MAY} have a name="\textbf{
 validity\_predictor}" with datatype="char" and arraysize="*" with an
-identifier of the software used to calculate the visibility.}
+identifier of the software used to calculate the observability.}
 
 \item {Exactly one field \textbf{MUST} have a name="\textbf{t\_start}" with\\
 utype="\textbf{Char.TimeAxis.Coverage.Bounds.Limits.StartTime}"\\ with
-datatype="float",  ucd="time.start", unit="d" containing the start of the visibility period.}
+datatype="float",  ucd="time.start", unit="d" containing the start of the observability period.}
 
 \item {Exactly one field \textbf{MUST} have a name="\textbf{t\_stop}" with\\
 utype="\textbf{Char.TimeAxis.Coverage.Bounds.Limits.StopTime}"\\, with
-datatype="float", ucd="time.end" and unit="d" containing the end of the visibility period.}
+datatype="float", ucd="time.end" and unit="d" containing the end of the observability period.}
 
-\item {Exactly one field \textbf{MUST} have a name="\textbf{t\_visibility}"
+\item {Exactly one field \textbf{MUST} have a name="\textbf{t\_observability}"
```

Comment:
This column name change should also be mentioned in the Change Log.

I guess the existing ObjVisSAP/ObjObsSAP service(s) MUST be updated too, isn't it?

Reply:
Yes.

Reply:
added to the Change log